docs/rancher/harvester-ui-extension.md

```diff
@@ -0,0 +1,104 @@
+</head>
+
+Start from Rancher 2.10.0, Harvester integration is provided by [Harvester UI Extension](https://github.com/harvester/harvester-ui-extension). Harvester UI Extension is a standard Rancher extension that built on [@rancher/shell](https://www.npmjs.com/package/@rancher/shell) library.
+
+
+Original [Harvester dashboard](https://github.com/harvester/dashboard) only keeps to maintain Harvester v1.3.x and v1.4.x releases. New features in future release will be implemented in Harvester UI Extension.
+
+
+
+
+## Support Matrix
+
+Harvester UI Extension version will align with Harvester release version since v1.5.0, 
+
+:::caution
+Harvester UI Extension only supports Harvester cluster versions greater or equal to 1.3.0.
+:::
+
+
+| Harvester Cluster         | Harvester UI Extension          | Minimum supported Rancher  |
+| --------------------------| ------------------------------- | ------------------------   |
+| v1.3.0 / v1.3.1 / v1.3.2  | v1.0.2                          | Rancher 2.10.1             |
+| v1.4.1                    | v1.0.3                          | Rancher 2.10.1             |
+| v1.4.2                    | v1.0.4                          | Rancher 2.10.1             |
+| v1.5.0                    | v1.5.0                          | Rancher 2.11.0             |
+
+
+## Install on Rancher 2.10
+
+:::caution
+Rancher 2.8 and 2.9 do not support to install Harvester UI Extension, only Rancher 2.10.0 and later version support
+:::
+
+
+### Automatically Install
+
+We provide `auto install` process to install Harvester UI Extension with one button click.
+
+1. On the Rancher UI, go to **Virtualization Management** screen.
+
+1. Click `Install` button and allow some time for the extension to be installed, and the page will auto refresh once the installation process is done. Don't navigate to other pages while in installation process, otherwise the auto process will fail. 
+
+ ![](/img/v1.5/rancher/auto-install-ui-extension.png)
+
+
+### Manually Install
+If auto installation process is failed, you can also setup Harvester UI Extension repository URL and install it manually.
+ 
+1. On the Rancher UI, go to **local > Apps > Repositories**.
+
+1. Click **Create** button, give a repository name and choose the second option `Git repository containing Helm chart or cluster template definitions`.
+
+  Fill in the following properties:
+    - Git Repo URL: **https://github.com/harvester/harvester-ui-extension**
+    - Git Branch: **gh-pages**
+  
+  ![](/img/v1.5/rancher/ui-extension-app-repository-setup.png)
+
+1. Go to the **Extensions** screen, click **install** in Available tab.
+
+  ![](/img/v1.5/rancher/ui-extension-available-tab.png)
+
+
+## Upgrade Harvester UI Extension
+
+### Automatically Upgrade
+1. On the Rancher UI, go to the **Virtualization Management**.
+
+1. Click `Update` button and allow some time for the extension to be upgraded, and the page will auto refresh once the upgrade process is done.
```

Comment:
Please mention that the Upgrade button will be visible when a new Extension version is available